File: python/class 3/question 5.py
_3_digit_num = int(input("Enter a 3 digit number: "))

sum_of_digits = 0

for digit in str(_3_digit_num):
  sum_of_digits += int(digit)
  
print(sum_of_digits)



#alternatively you could do 
_3_digit_num = int(input("Enter a 3 digit number: "))
print(sum([int(_) for _ in str(_3_digit_num)]))

File: java/class 3/Question5.java
import java.util.Scanner;

public class Question5{
  public static void main(String[] args){
    Scanner input = new Scanner(System.in);
    System.out.print("Enter a 3 digit number: ");
    int num = input.nextInt();
    input.close();
    int sum = 0;
    while(num > 0){
      sum += num % 10;
      num /= 10;
    }
    System.out.println("The sum of the digits is: " + sum);
  }
}
